### ISSUE TITLE
Tailwindcss ignores classes from node_modules

### ISSUE THREAD

**Issue by Napishite** [rails/tailwindcss-rails#232]
Hi!

I found case when Tailwindcss doesn't build overriden classes from node module on assets:precompile during deploy.

**TL;DR:** 

If you have in your Gemfile tailwindcss-rails *before* jsbundling-rails tailwind will not scan project dependencies for styles in production environment during assets:precompile. If jsbundling-rails goes first it works well.

**Possible solution:**

This happen because they both enchasing  assets:precompile task and tailwindcss-rails run tailwindcss:build *before* javascript dependencies will be installed by javascript:build and tailwind just have nothing to scan.

Maybe it could be solved by mounting tailwindcss:build to  javascript:build if it defined? 

```
# build.rake:

if Rake::Task.task_defined?("javascript:build")
  Rake::Task["javascript:build"].enhance(["tailwindcss:build"])
else
  Rake::Task["assets:precompile"].enhance(["tailwindcss:build"])
end
```
________

**More details about my case:**

In my app I use Tailwindcss-rails, [jsbundling-rails](https://github.com/rails/jsbundling-rails), with esbuild and fullcalendar.

I wanted to override some classes and added to my application.tailwind.css somehting like this:

```
    .fc .fc-daygrid-day.fc-day-today {
        background-color: rgba(107,33,168,0.15);
    }
```

And also set path fullcalendar in my tailwind.config.js:

```
  content: [
    ...
    "./node_modules/@fullcalendar/**/*.js"
  ],
```

Overriden classes worked perfectly in dev environment, because I aways have node_modules installed at the moment of running tailwindcss:build. But in production (in my case staging) environment when I run assets:precompile tailwindcss generated all styles excepting related to fullcalendar.

**Comment by flavorjones** [comment]
Apologies for not responding before now. I'd welcome a PR that either documents this rake task workaround or updates the rake tasks for this use case.
